database/migrations/2014_10_12_000000_create_users_table.php

```diff
@@ -14,7 +14,9 @@ public function up(): void
         Schema::create('users', function (Blueprint $table) {
             $table->id();
             $table->string('name');
+            $table->string('username');
             $table->string('email')->unique();
+            $table->date('date_of_birth');
```

Comment:
Please don't modify the core migration file. Create new migration to add new columns.